include/deal.II/particles/generators.h

```diff
@@ -162,6 +165,86 @@ namespace Particles
         StaticMappingQ1<dim, spacedim>::mapping,
       const unsigned int random_number_seed = 5432);
 
+
+    /**
+     * A function that generates particles at the locations of the support
+     * points of a DoFHandler, possibly based on a different Triangulation with
+     * respect to the one used to construct the ParticleHandler.
+     * The total number of particles that is added to the @p particle_handler object is
+     * the number of dofs of the DoFHandler that is passed that are within the
+     * triangulation and whose components are within the ComponentMask.
+     * This function uses insert_global_particles and consequently may induce
+     * considerable mpi communication overhead.
+     *
+     * @param[in] a DOF handler that may live on another triangulation that is
```

Comment:
You'll have to name the argument:
```suggestion
     * @param[in] dof_handler A DOF handler that may live on another triangulation that is
```